types/pdfmake/index.d.ts

```diff
@@ -171,16 +181,24 @@ declare module 'pdfmake/build/pdfmake' {
     }
 
     interface TDocumentDefinitions {
-        info?: TDocumentInformation;
+        background?: (currentPage?: any, pageSize?: any) => any;
```

Comment:
This could also be a `string` and the function returns a `string` as well. The arguments are up to the developer and irrelevant for the internals of pdfmake (as far as I understand the current docs).

```
background?: () => string | string;
```

Reply:
ok